constants/appLinks.ts

```diff
@@ -57,6 +57,79 @@ const IBM_Q_EXPERIENCE: GeneralLink = {
   label: 'IBM Quantum Lab'
 }
 
+interface Prerequisite {
+  title: string;
+  description: string;
+  segment: CtaClickedEventProp;
+  url: string;
+}
+
+const introductionCoursePrerequisites: Prerequisite[] = [
```

Comment:
I think that it's better to have this `introductionCoursePrerequisites` on the `introduction-course` page